pkg/helm/actions/config.go

```diff
@@ -10,13 +10,19 @@ import (
 	"k8s.io/klog"
 )
 
-var settings = cli.New()
+var settings = initSettings()
```

Comment:
Curious, `settings` is not used elsewhere in the file.  Why does this happen here?

Reply:
Is `settings` a good name for this var?  
Not a fan of globals, but it did sneak past in a previous review.  "settings" does sound like it could be anything.  Per:
https://github.com/pedjak/console/blob/cdf6b189b71e488033ecaba7d90258d9f9453478/pkg/helm/actions/install_chart.go#L18
One could wonder what it is and where its created.

Reply:
sure, I agree with you... but I would avoid making this PR bigger than needed to fix the discovered bug. Let's explore it in a separate PR, after we merge this?

Reply:
I would rather handle the error now that we know we should.

Reply:
Even just logging is fine.